models/model.js

```diff
@@ -6,3 +6,17 @@ exports.fetchCategories = () => {
     return result.rows;
   });
 };
+
+exports.fetchReviewId = (review_id) => {
+  return db
+    .query(`SELECT * FROM reviews WHERE review_id = $1;`, [review_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return promise.reject({
```

Comment:
A slight mistake here... promise should be `Promise` with a capital P.

Reply:
Really good to see you've got logic here to deal with an empty array and what you want do with it